src/meetup_manager/meetup_manager/parser.py

```diff
@@ -0,0 +1,41 @@
+import json
+import re
+
+
+def parser(issue: str):
```

Comment:
Did we test it works in al cases?

Very rare case, but what if a speaker intentionally leaves out a section intentionally? What happens then?

Need to add more error handling

Reply:
I have tested manually on test cases that i can think of.
If speaker leaves out a section, then value corresponding to that section will be empty. I will add tests, that will clear a lot of things.